layers/meta-balena-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging/0002-env_default.h-Add-support-for-OS_KERNEL_CMDLINE.patch

```diff
@@ -0,0 +1,77 @@
+Index: git/include/environment/ti/mmc.h
```

Comment:
This patch seems to be removed from SRC_URI in the next commit, so it does not seem to be used. Can it be removed from this commit then?

Reply:
This seems like it's still added here but then deleted in the next commit: https://github.com/balena-os/balena-beaglebone/pull/662/commits/2178ff894159a301265aee4b5051c7249967521e

Can we just not add it at all here?

Reply:
That is correct, because the first commit belongs to the other PR never merged, updating u-boot-ti-staging, not u-boot-bb.org. After that commit I updated meta-ti so the patch went away and finally the last commit is the only one which belongs here.

If you want I can try and just keep the last commit, but I would definitely keep the old ones in case the layer can *finally* use meta-ti's default u-boot, u-boot-ti-staging